features/aa/aa_features/orphans.feature

```diff
@@ -5,9 +5,9 @@ Feature:
 
   Background:
     Given the following orphans exist:
-      | name     | father_given_name | family_name | spon_status | death_date | mother   | birth_date | contact   | o_city  | o_province | o_hood  | c_city  | c_province | c_hood  |
```

Comment:
Sponsorship status is managed by the application and should never be specified explicitly, otherwise it creates logical inconsistencies. e.g. status is `sponsored` but there is no `current_sponsor`, which breaks the views.